src/app/Components/Navbar.tsx

```diff
@@ -0,0 +1,124 @@
+import MenuIcon from '@mui/icons-material/Menu';
+import {
+	AppBar,
+	Box,
+	Button,
+	Drawer,
+	Grid2,
+	IconButton,
+	List,
+	ListItem,
+	ListItemButton,
+	ListItemText,
+	Stack,
+	Toolbar,
+	Typography,
+} from '@mui/material';
+import Image from 'next/image';
+import React, { useState } from 'react';
+import LogoImage from '../../assets/images/mainLogo.png';
+
+export const DesktopNavbar = () => {
+	return (
+		<Box sx={{ flexGrow: 1 }}>
+			<AppBar position="static">
+				<Toolbar>
+					<Stack
+						direction="row"
+						sx={{
+							justifyContent: 'space-between',
+							alignItems: 'center',
+							width: '100%',
+						}}
+						spacing={2}
+					>
+						<Box>
+							<IconButton
+								size="large"
+								edge="start"
+								color="inherit"
+								aria-label="menu"
+								sx={{ mr: 2 }}
+							>
+								<Image
+									src={LogoImage}
+									height={56}
+									width={56}
+									alt="Logo for Koalaty Speech and Feeding."
+								/>
+							</IconButton>
+						</Box>
+						<Typography variant="h6">
+							Koalaty Speech and Feeding Therapy Services
+						</Typography>
+						<Stack direction="row" spacing={1}>
```

Comment:
We can probably reuse List like we are in the mobile version and make the nav items global to both components since they're being reused.
```suggestion
<List sx={{ display: 'flex' }}>
	{navItems.map((item) => (
		<ListItem key={item} disablePadding>
			<ListItemButton sx={{ p: 1 }}>
				<ListItemText primary={text} />
			</ListItemButton>
		</ListItem>
	))}
</List>
```